### PR TITLE
[MTDSA-23138]  Changelog entry for the new APIs created as part of monolith breakup of individuals-income-received-api

### PR DESCRIPTION
**Pull request by sudoku007**
Changelog entry for the new APIs created as part of monolith breakup of individuals-income-received-api

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +62,42 @@ The following table shows which APIs are linked to which forms. To download the
 Please note that the date shown is the date that the change was released to the Sandbox or Production.
 
 ---
+### 9 April 2024
+
+The following 8 new APIs have been created and now available in Sandbox as part of the breakup of `individuals-income-received-api`:
+
```
Comment (fmcgrath) [inline README.md:68]: and are now available?

**Review by fmcgrath — COMMENTED** [review]
Review wording at start of changelog entry.

File: README.md (hunk 1 above)
Comment (ben-HMRC) [reply README.md:69]: Agree with the above!

File: README.md [hunk 2]
```diff
@@ -62,6 +62,45 @@ The following table shows which APIs are linked to which forms. To download the
...
+#### individuals-employments-income-api
+
+This API allows developers to retrieve, create, amend or delete data relating to
+- `employment income`
+- `other employment income`
+- `Non-PAYE employment income`
+
+#### individuals-dividends-income-api
+
+This API allows developers to retrieve, create, amend or delete data relating to `dividends income`.
```
Comment (ben-HMRC) [inline README.md:79]: Change 'allows developers' to 'enables you' for all API descriptions

File: README.md [hunk 3]
```diff
@@ -62,6 +62,45 @@ The following table shows which APIs are linked to which forms. To download the
...
+
+#### individuals-foreign-income-api
+
+This API allows developers to retrieve, create, amend or delete data relating to `foreign income`.
+
```
Comment (ben-HMRC) [inline README.md:84]: Why are 'foreign income', 'insurance policies income' etc formatted as code with backticks? No need to do that

Comment (sudoku007) [reply README.md:84]: Removed the backticks

File: README.md [hunk 4]
```diff
@@ -62,6 +62,45 @@ The following table shows which APIs are linked to which forms. To download the
 Please note that the date shown is the date that the change was released to the Sandbox or Production.
 
 ---
+### 9 April 2024
+
+The following 8 new APIs have been created and are now available in Sandbox as part of the breakup of 
+individuals-income-received-api:
```
Comment (fmcgrath) [inline README.md:68]: I would write the intro as follows:

As part of the breakup of [individuals-income-received-api](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/individuals-income-received-api), the following APIs are available in Sandbox.

**Review by fmcgrath — CHANGES_REQUESTED** [review]
Two minor wording issues.

File: README.md (hunk 4 above)
Comment (sudoku007) [reply README.md:68]: in Sandbox or in the Sandbox?

File: README.md [hunk 5]
```diff
@@ -62,6 +62,45 @@ The following table shows which APIs are linked to which forms. To download the
...
+
+#### individuals-employments-income-api
+
+This API enables you to retrieve, create, amend or delete data relating to
+- employment income
+- other employment income
+- Non-PAYE employment income
```
[the remark this answers is not in the document]
Comment (sudoku007) [reply README.md]: updated to use same capitalisation

File: README.md (hunk 4 above)
Comment (sudoku007) [reply README.md]: Updated as suggested

**Review by fmcgrath — CHANGES_REQUESTED** [review]
One minor change needed.

File: README.md [hunk 6]
```diff
@@ -62,6 +62,44 @@ The following table shows which APIs are linked to which forms. To download the
 Please note that the date shown is the date that the change was released to the Sandbox or Production.
 
 ---
+### 9 April 2024
+
+As part of the breakup of individuals-income-received-api, the following APIs are available in Sandbox.:
+
```
[the remark this answers is not in the document]
Comment (sudoku007) [reply README.md:68]: Updated

**Review by fmcgrath — APPROVED** [review]

**Pull request merged**